File: 04_Communication/MATLAB/LedOnOff.m
s = serial('/dev/tty.usbserial-DN0421L4', 'Baudrate', 9600);
fopen(s);
for i = 1:5
    fprintf('Turnning LED ON\n');
    fprintf(s, 'ON');
    pause(2);
    fprintf('Turnning LED OFF\n');
    fprintf(s, 'OFF');
    pause(2);
end
fclose(s);
fprintf('Done!\n');

File: 04_Communication/Python/LedOnOff.py
import serial
import time

ser = serial.Serial('/dev/tty.usbserial-DN0421L4')  # open serial port
print(ser.name)         # check which port was really used
for k in range(5):
    print("ON")
    ser.write(b'ON\n')
    time.sleep(1)
    print("OFF")
    ser.write(b'OFF\n')
    time.sleep(1)
ser.close()             # close port
print("Done")